caiman/components_evaluation.py

```diff
@@ -13,9 +13,10 @@
 from typing import Any, Union
 import warnings
 
+import caiman
 from caiman.paths import caiman_datadir
-from .utils.stats import mode_robust, mode_robust_fast
-from .utils.utils import load_graph
+import caiman.utils.stats
```

Comment:
Generally I think `caiman.utils` I would just import `stats` so instead of having to read `caiman.utils.stats` later I just have to read `stats.mode_robust` later.